core/src/trezor/pin.py

```
@@ -6,85 +6,87 @@
 
     if callable(keepalive_callback):
         keepalive_callback()
 
-    if message is not None:
-        text = ""
-        if message == "read fp data":
-            text = _(i18n_keys.MSG__READING_FINGERPRINT_FROM_SECURITY_CHIP)
+    _scr_need_create = _scr is None
+    if _scr_need_create:
+        try:
+            _scr.subtitle
```

Ruff (0.11.9)

Found useless expression. Either assign it to a variable or remove it.

(B018)